tests.toml

```
@@ -8,11 +8,11 @@ test_format = 1.0
     # Commits to test upgrade from
     # -------------------------------
 
-    test_upgrade_from.c5cf91ad.name = "Upgrade from 25.0.2"
-    test_upgrade_from.caf917f3.name = "Upgrade from 26.0.2"
+    #test_upgrade_from.c5cf91ad.name = "Upgrade from 25.0.2"
+    #test_upgrade_from.caf917f3.name = "Upgrade from 26.0.2"
```

```suggestion
    test_upgrade_from.c5cf91ad.name = "Upgrade from 25.0.2"
    test_upgrade_from.caf917f3.name = "Upgrade from 26.0.2"
```

I'm thinking we should add these back now that CI is speedy again ?